src/citrine/builders/predictors.py

```diff
@@ -157,7 +157,11 @@ def build_simple_ml(
     latent_variables: List[Descriptor],
     training_data: Optional[List[DataSource]] = None
 ) -> GraphPredictor:
-    """Build a graph predictor that connects all inputs to all latent variables and outputs.
+    """
+    Build a graph predictor that connects all inputs to all latent variables and outputs.
+
+    The structure of the returned, unregistered graph predictor should be analogous to the
```

Comment:
We should state definitively that it _will_ produce the same results as a SimpleMLPredictor. So just replace "should be" with "will be".